DoSomething/gambit
# Adds v2 namespace for bots

#### What's this PR do?
* Changes URL to `/v2/bots/donorschoose` to use for our Science Sleuth Donation mData's this year to better document breaking changes from v1 routes removed in #573
* Removes outdated README. One of my todos is to add a documentation folder a la [Northstar](https://github.com/DoSomething/northstar/tree/dev/documentation) and [Phoenix](https://github.com/DoSomething/phoenix/tree/dev/documentation), but for schema we've been updating the properties of config schema as inline comments in the relevant `models` directory.
* Returns a 500 error if our `smsConfigsLoader` can't find the required `config` document corresponding to our DonorsChoose Moco Campaign or DonorsChoose bot set as environment configuration variables.


#### How should this be reviewed?
* Verify new URL works
* Set `DONORSCHOOSE_MOCO_CAMPAIGN_ID` and `DONORSCHOOSE_BOT_ID` to values that don't exist to confirm error code 500 is returned.

#### Relevant tickets
Refs finalized URLs for #539 mData's

#### Checklist
